src/croniter/croniter.py

```diff
@@ -617,13 +647,14 @@ def _get_prev_nearest(self, x, to_check):
 
     def _get_next_nearest_diff(self, x, to_check, range_val):
         for i, d in enumerate(to_check):
-            if d == "l":
+            if d == "l" and range_val is not None:
```

Comment:
Add comment for use.

Reply:
I can do it myself but I prefer you to do it for you to keep all this big PR paternity :)

I propose you the bellow modification than you can take mostly as-is (untested, verify tests :)):
- I moved a part of the docstring to where it is effectively used.
- I modified a bit the comment.
- I added a final explicit `return None`

Thus, this enhances code readability, while doing the same afterall.

```python
    def _get_next_nearest_diff(self, x, to_check, range_val):
        """
        `range_val` is the range of a field.
        If no available time, we can move to next loop(like next month).
        `range_val` can also be set to `None` to indicate that there is no loop.
        Currently, it is only used for `year` field        
        """
        for i, d in enumerate(to_check):
            if d == "l" and range_val is not None:
                # if 'l' then it is the last day of month
                # => its value of range_val
                d = range_val
            if d >= x:
                return d - x
        # When range_val is None and x not exists in to_check,
        # `None` will be returned to suggest no more available time
        if range_val is None:
             return None
        return to_check[0] - x + range_val
```